std/json.joke

```diff
@@ -21,7 +21,11 @@
 
 (defn json-seq
   "Returns the json records from rdr as a lazy sequence.
-  rdr must be a string or implement io.Reader."
-  {:added "1.0"
-  :go "jsonSeqOpts(rdr)"}
-  [^Object rdr])
\ No newline at end of file
+  rdr must be a string or implement io.Reader.
+  Optional opts map may have the following keys:
+  :keywords? - if true, JSON keys will be converted from strings to keywords."
+  {:added "0.18.0"
```

Comment:
Let's make it `1.0`, joker is still in 0.x version so the next stable one will be 1.0.